mne_qt_browser/tests/test_pg_specific.py

```diff
@@ -135,16 +133,35 @@ def test_ch_specific_annot(raw_orig, pg_backend):
     # change the stop value of the annotation
     annot_dock.stop_bx.setValue(6)
     annot_dock.stop_bx.editingFinished.emit()
-    # does the channel specific rectangle stay in sync with the annot?
+    # does the single channel annot stay within the annot
     assert annot_dock.stop_bx.value() == 6
-    assert last_fill_between.xData[1] == 6
+    assert single_channel_annot.lower.xData[1] == 6
 
     # now change the start value of the annotation
     annot_dock.start_bx.setValue(4)
     annot_dock.start_bx.editingFinished.emit()
     # does the channel specific rectangle stay in sync with the annot?
     assert annot_dock.start_bx.value() == 4
-    assert last_fill_between.xData[0] == 4
+    assert single_channel_annot.lower.xData[0] == 4
+
+    # test if shift click an existing annotation removes object
+    ch_index = np.mean(annot.single_channel_annots["MEG 0133"].ypos).astype(int)
+    fig._fake_click(
+        (4 + 2 / 2, ch_index),
+        xform="data",
+        button=1,
+        modifier=Qt.ShiftModifier,
+    )
+    assert "MEG 0133" not in annot.single_channel_annots.keys()
+
+    # test if shift click on channel adds annotation
```

Comment:
> @nmarkowitz Failure is related to the changes:

Indeed https://github.com/mne-tools/mne-python/pull/12669 hasn't landed and even once it does the `MNE maint/*` tests will fail (which I'll update to 1.7 momentarily). The best option here I think would be to take the new tests and split them into a new test function like
```
def test_ch_specific_annot_interaction
```
(and maybe rename the existing one `..._annot_display` or something). Then you can decorate the new function like:
```
@pytest.mark.skipif(not hasattr(mne.something.SomeClass, "_toggle_single_channel_annotation"), reason="Needs MNE 1.8+")
def test_...
```
or similar. Then in this test suite should pass, and once https://github.com/mne-tools/mne-python/pull/12669 lands tests will run on the `main` version of the CIs and tests should be skipped on the `maint/*` version.

Make sense?

Reply:
I think so. I'll keep trying and see how far I can get